streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -3447,4 +3449,51 @@ void process(final ProcessorSupplier<? super K, ? super V> processorSupplier,
     void process(final ProcessorSupplier<? super K, ? super V> processorSupplier,
                  final Named named,
                  final String... stateStoreNames);
+
```

Comment:
Just a personal request: we try to keep the method somewhat "ordered" base on their semantics (stateless: no-side effect, with side effect, othes; stateful: grouping/aggregating/join; last PAPI integration). Hence, I think it would be best to not add the methods at the end, but after the `to()` overloads and before `groupBy(...)` -- please use the same order in `KStreamImpl` to keep both classes aligned -- it make is easier to navigate through the code.